cmd/root.go

```diff
@@ -191,16 +224,20 @@ var RootCmd = &cobra.Command{
 		rootCmdCfg.BindPFlags(cmd.Flags())
 		log = newLogger("issue-sync", rootCmdCfg.GetString("log-level"))
 	},
+	// RunE is the main entry point of the application. It validates the
```

Comment:
You should not document the field; that is a property and concern of the cobra API. You may document the function value.